packages/core/src/stylable-value-parsers.ts

```diff
@@ -83,13 +85,17 @@ export const SBTypesParsers = {
     '-st-root'(value: string) {
         return value === 'false' ? false : true;
     },
-    '-st-global'(decl: postcss.Declaration, _diagnostics: Diagnostics) {
-        // Experimental
+    '-st-global'(decl: postcss.Declaration, diagnostics: Diagnostics): SelectorNodes | undefined {
         const selector = parseSelectorWithCache(
             decl.value.replace(/^['"]/, '').replace(/['"]$/, ''),
             { clone: true }
         );
-        // ToDo: handle or warn on multiple selectors
+        if (!selector[0]) {
+            diagnostics.error(decl, CSSClass.diagnostics.EMPTY_ST_GLOBAL());
+            return;
+        } else if (selector.length > 1) {
+            diagnostics.error(decl, CSSClass.diagnostics.UNSUPPORTED_MULTI_SELECTORS_ST_GLOBAL());
```

Comment:
should be info

Reply:
discussed this with Tom who thinks that both should be error for now - what do you think?

Reply:
changed to error with no transformation in case of multiple selectors. We can add support when `:is` is enabled in the future.